spec/unit/tariff_synchronizer/taric_update_spec.rb

```diff
@@ -173,6 +173,20 @@
     end
   end
 
+  describe '#next_update' do
+    subject(:next_update) { build(:taric_update, :applied, example_date: Date.parse('2022-08-25')).next_update }
+
+    it { is_expected.to have_attributes(issue_date: Time.zone.today, filename: '2022-08-26_TGB22238.xml') }
```

Comment:
Is this test going to pass tomorrow 🤔 ?
(maybe, I just don't trust much test with variable time)

Reply:
It will now, yah